assets/js/editor-components/incompatible-extension-notice/editor.scss

```diff
@@ -18,11 +18,11 @@
 			width: 24px;
 			height: 24px;
 			margin-right: 6px;
-			min-width: max-content;
+			min-width: 24px; // Ensure that notice is visible in Safari. See https://github.com/woocommerce/woocommerce-blocks/issues/11734
```

Comment:
considering this is min width and the settings editor is a defined with it's ok to leave in this fixed value